src/discord_bot/guild_storage.rs

```diff
@@ -44,6 +44,12 @@ pub struct GuildStorage {
     pub users_sent_to_support: HashSet<UserId>,
     #[serde(default)]
     pub counters: HashMap<String, u64>,
+    #[serde(default)]
+    pub octal_counter_channel: Option<ChannelId>,
+    #[serde(default)]
+    pub octal_counter: i32,
+    #[serde(default)]
+    pub octal_counter_latest_user: Option<UserId>,
```

Comment:
Could these be moved into their own struct perhaps? Like we have for `ChessState` and `RoleData`